evergreen.yml

```
@@ -248,3 +274,4 @@ buildvariants:
       - name: compile
       - name: sign
       - name: release
+      - name: downloadcenter-update
```

Super small nit: Would be clearer for me with a dash
```suggestion
      - name: download-center-update
```